src/emuvim/api/sonata/dummygatekeeper.py

```diff
@@ -74,7 +75,13 @@
 BIDIRECTIONAL_CHAIN = False
 
 # override the management interfaces in the descriptors with default docker0 interfaces in the containers
-USE_DOCKER_MGMT = True
+USE_DOCKER_MGMT = False
```

Comment:
Can we change this back to its original value? It might break a lot of other stuff done with the emulator. It should only be deactivated for the particular demo where it is needed.